src/pytest_plugins/consume/hive_simulators/conftest.py

```diff
@@ -55,7 +71,10 @@ def eest_consume_commands(
     Commands to run the test within EEST using a hive dev back-end.
     """
     hive_dev = f"./hive --dev --client-file configs/develop.yaml --client {client_type.name}"
-    consume = f'consume {test_suite_name.split("-")[-1]} -v --latest -k "{test_case.id}"'
+    consume = (
+        f'consume {test_suite_name.split("-")[-1]} -v --input "latest-develop-release" -k '
```

Comment:
This is used to display the consume command in order to reproduce the test locally, right? Here `"latest-develop-release"` will be ambiguous and could point to a different release than the one executed in this instance.